### ISSUE TITLE
Setup dev/test seed data for user roles in a school

### ISSUE THREAD

**Issue by sra405** [RaspberryPiFoundation/editor-api#291]
Once we model schools (already implemented) and roles in `editor-api` we can seed some data for local dev and testing purposes. More than happy for this to be altered (but probably worth documenting in the README as part of this ticket) but currently proposing:

- [x] - Adding a school with `jane.doe@example.com` as the user who submitted the request
- [x] - Adding `jane.doe@example.com` as an owner and teacher of that same school
- [ ] - ~~Adding `john.doe@example.com` as a student of that same school~~

### Notes

In the not too distant future having a full Pi account holder as a student may not work (students will be a separate table and obtained via and different endpoint in profile) but as long as the implementation in simple to change then we can update that as we go

**Comment by sra405** [comment]
currently blocked by https://github.com/RaspberryPiFoundation/editor-api/issues/285

**Comment by danhalson** [comment]
https://github.com/RaspberryPiFoundation/profile/pull/1627/files has introduced some seeded students to profile, but since the student api integration one editor api is still incomplete this work is blocked and that has been descoped
